tardis/transport/montecarlo/tests/test_tracker_utils.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import numpy as np
+from numba import typeof
+
+from tardis.transport.montecarlo.packet_trackers import (
+    RPacketTracker,
+    RPacketLastInteractionTracker,
+    generate_rpacket_tracker_list,
+    generate_rpacket_last_interaction_tracker_list,
+)
+
+
+def test_generate_rpacket_tracker_list():
+    no_of_packets = 10
+    length = 10
+    random_index = np.random.randint(0, no_of_packets)
+
+    rpacket_tracker_list = generate_rpacket_tracker_list(no_of_packets, length)
+
+    assert len(rpacket_tracker_list) == no_of_packets
+    assert len(rpacket_tracker_list[random_index].shell_id) == length
+    assert typeof(rpacket_tracker_list[random_index]) == typeof(
+        RPacketTracker(length)
+    )
+
+
+def test_generate_rpacket_last_interaction_tracker_list():
+    no_of_packets = 50
+    random_index = np.random.randint(0, no_of_packets)
```

Comment:
Regarding the time usage of the random number generator
```python
import numpy as np
```


```python
no_of_packets = 50
```


```python
%timeit np.random.randint(0, no_of_packets)
```

    2.89 μs ± 199 ns per loop (mean ± std. dev. of 7 runs, 100,000 loops each)



```python

```